tests/js/spec/components/__snapshots__/confirmDelete.spec.jsx.snap

```diff
@@ -79,8 +79,6 @@ exports[`ConfirmDelete renders 1`] = `
             >
               <Component
                 className="css-8yn43f-FieldWrapper-getPadding-borderStyle-inlineStyle etqqcs20"
-                hasControlState={false}
-                inline={false}
```

Comment:
This disappear because `shouldForwardProp` takes them off the styled `Component`.